application/src/components/project-component/ProjectCard.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <div class="card" @click="redirectOnClick()">
+        <div class="picture" :style="{ 'background-image': 'url(\'' + projectData.image + '\')' }"/>
+        <div class="description">
+            <p :class="[GetStatusClass, 'status']">{{GetStatusText}}</p>
+            <h4>{{projectData.name}}</h4>
+            <p>By {{projectData.owner.name}}</p>
+            <p>Launched {{projectData.created_on}}</p>
+            <p class="description-text">{{projectData.introduction}}</p>
+            <p><font-awesome-icon icon="globe-americas"/> {{projectData.location}}</p>
+            <p class="tag">{{projectData.tag}}</p>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+  name: 'ProjectCard',
+  props: {
+    projectData: {
+      required: true,
+      type: Object
+    }
+  },
+  methods: {
+    redirectOnClick () {
+      console.log('getting to project')
```

Comment:
```suggestion
      window.location = `/project/${this.projectData.id}`
```

I think this should work now